builtin/providers/aws/resource_aws_db_parameter_group_test.go

```diff
@@ -193,6 +249,16 @@ func testAccCheckAWSDBParameterGroupExists(n string, v *rds.DBParameterGroup) re
 	}
 }
 
+func RandomString(strlen int) string {
```

Comment:
small nitpick: if this function isn't used outside of this file/package, it should be lowercased and not exported